core/arch/arm/plat-imx/imx6.c

```diff
@@ -45,9 +45,9 @@ void plat_cpu_reset_late(void)
 		/* primary core */
 #if defined(CFG_BOOT_SYNC_CPU)
 		/* set secondary entry address and release core */
-		write32(CFG_TEE_LOAD_ADDR, SRC_BASE + SRC_GPR1 + 8);
-		write32(CFG_TEE_LOAD_ADDR, SRC_BASE + SRC_GPR1 + 16);
-		write32(CFG_TEE_LOAD_ADDR, SRC_BASE + SRC_GPR1 + 24);
+		write32(TEE_TEXT_VA_START, SRC_BASE + SRC_GPR1 + 8);
+		write32(TEE_TEXT_VA_START, SRC_BASE + SRC_GPR1 + 16);
+		write32(TEE_TEXT_VA_START, SRC_BASE + SRC_GPR1 + 24);
```

Comment:
these should write a physical address.
Either a virt2phys, or a comment stating this expects flat mapping.

Reply:
ok